testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/ra/AddAdminObjectToRAOnlineTestCase.java

```diff
@@ -26,6 +29,7 @@
     @Before
     public void connect() throws IOException {
         client = ManagementClient.online(OnlineOptions.standalone().localDefault().build());
+        Assume.assumeTrue(client.version().greaterThanOrEqualTo(ServerVersion.VERSION_2_1_0));
```

Comment:
Hm.... It seems that WF8.1 doesn't contain the fix... I'll investigate this further